src/app/models/tech-record.model.ts

```diff
@@ -25,6 +25,14 @@ export interface Plate {
   plateIssuer: string;
 }
 
+export interface AuthoIntoService {
```

Comment:
We should AuthorisationIntoService

Reply:
The naming convention is kept similar to what exists on the API spec.